internal/pkg/mojang/profile.go

```diff
@@ -0,0 +1,128 @@
+package mojang
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"net/url"
+)
+
+func isURL(s string) bool {
+	u, err := url.ParseRequestURI(s)
+	return err == nil && (u.Scheme == "http" || u.Scheme == "https")
+}
+
+func (c *Client) ProfileInformation(ctx context.Context, accountToken string) (*ProfileInformationResponse, error) {
+
+	headers := http.Header{}
+
+	headers.Set("Authorization", "Bearer "+accountToken)
```

Comment:
should take accountToken as a param (optional if needed) to the client itself rather than passing it around everywhere